src/main/kotlin/edu/cwru/raycaster/Vector.kt

```diff
@@ -56,8 +61,8 @@ interface VectorOperations<T> where T : Comparable<T>, T : Number {
 interface VectorOperationsInt : VectorOperations<Int> {
   override fun plus(vec: Vec2<Int>) = Vec2Int(x + vec.x, y + vec.y)
   override fun minus(vec: Vec2<Int>) = Vec2Int(x - vec.x, y - vec.y)
-  override fun times(value: Int) = Vec2Int(x * value, y * value)
-  override fun div(value: Int) = Vec2Int(x / value, y / value)
```

Comment:
This is not how vector multiplication is defined.

Reply:
Did you mean cross product?

Reply:
I don't think vector multiplication exists, right? Just cross/dot products. Made sense to have elementwise multiplication for the * operator

Reply:
Huh? You undid my scalar multiplication?